### ISSUE TITLE
newly created "Akte" shows incorrect slug

### ISSUE THREAD

**Issue by textloop** [cceh/mea-atom-installation#67]
The following "Akte" was recently creaed by Maxime to group all letters written in 1933. 
Signatur: TI MEA-GGA-I-BW-1933a
Titel: Briefwechsel des Jahres 1933

The url shows an incorrect slug:
http://.../a/index.php/mea-gga-i-bw-1934

@gioele: is this something that can be corrected easily or is it a candidate for the list of future enhancements, to work on as soon as there is new funding?

**Comment by gioele** [comment]
This needs a quite intrusive change in the code, because the current code assumes that the pieces that constitute a signature are either a string OR a number, not a mixture of the two as `1933a` is.

**Comment by textloop** [comment]
Misunderstanding- sorry!  I added the "a" for testing purposes, after the incorrect slug showed up. Every new created "Akte" will show this problem although only a string is filled in as Kennung/signature.
See the following example: 
Akte 1934 shows slug mea-gga-i-bw-1936
http://.../a/index.php/mea-gga-i-bw-1936
or 
New record for Akte 1933 shows mea-gga-i-bw-1946. (I deleted the old record)
http://.../a/index.php/mea-gga-i-bw-1946

Another example would be: 
http://.../a/index.php/mea-lw-i-1
This is one of the records that we have imported, but I added a "G" to the signature for correct sorting. The "G" is not showing up in the url. 

Seems to me that the installation is defining these slugs once and never changes them later, it does not care if the signature has been changed in the meantime. Is there a trick to fool the installation? ;-)
